### PR TITLE
Feature/grab size of image

### PR DESCRIPTION
**Pull request by Skumring**

### CODE REVIEW

File: app/models/estimate_screen.rb [hunk 1]
```diff
@@ -20,11 +21,23 @@ class EstimateScreen < ApplicationRecord
...
   validates :estimate, presence: true
+  validates :height, presence: true, numericality: { greater_than_or_equal_to: 0 }
   validates :position, presence: true, numericality: { greater_than_or_equal_to: 1 }
   validates :screen, presence: true, if: -> { title.blank? }
   validates :title, presence: true, if: -> { screen.file.blank? }
+  validates :width, presence: true, numericality: { greater_than_or_equal_to: 0 }
   validate :lock_estimate
   
+  def calculated_height
+    max_width = 320
```
Comment (skliarov) [inline app/models/estimate_screen.rb]: @Skumring this one should be dynamic (accepted as parameter)

Comment (Skumring) [reply app/models/estimate_screen.rb]: @skliarov ok

File: app/views/estimates/comments.slim [hunk 2]
```diff
@@ -21,7 +21,7 @@ table.table.table-bordered.estimate-table style='min-width: 1800px;'
           
         td.p-1.estimate-screen-col rowspan=calculated_rowspan
           .m-1 = estimate_screen.title
-          = image_tag estimate_screen.screen.width_320.url, class: 'img-fluid' if estimate_screen && estimate_screen.screen?
+          = image_tag estimate_screen.screen.width_320.url, class: 'img-fluid', style: "height: #{estimate_screen.calculated_height}px;" if estimate_screen && estimate_screen.screen?
```
Comment (skliarov) [inline app/views/estimates/comments.slim]: @Skumring set both separately like this `height="42" width="42"`, not in "style" parameter.

File: db/migrate/20200311123045_add_measurements_columns_to_estimate_screens.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+class AddMeasurementsColumnsToEstimateScreens < ActiveRecord::Migration[5.2]
+  def up
+    add_column :estimate_screens, :height, :integer, default: 0, null: false
+    add_column :estimate_screens, :width, :integer, default: 0, null: false
+    
+    # Processing measurements for already uploaded screens
+    EstimateScreen.where.not(screen: nil).find_each do |estimate_screen|
```
Comment (skliarov) [inline db/migrate/20200311123045_add_measurements_columns_to_estimate_screens.rb]: Move this code to a separate task

**Pull request merged**